api-report/driver-utils.api.md

```diff
@@ -317,6 +333,12 @@ export function streamFromMessages(messagesArg: Promise<ISequencedDocumentMessag
 // @public (undocumented)
 export function streamObserver<T>(stream: IStream<T>, handler: (value: IStreamResult<T>) => void): IStream<T>;
 
+// @public (undocumented)
+export enum SummaryCompressionProcessor {
+    // (undocumented)
+    SummaryBlob = 1
+}
```

Comment:
@milanro - assuming we can get the 'Bx' prefix approach working, I think we're ready to commit to a single strategy and can remove the 'SummaryCompressionProcessor' enum and config option.